tslint.json

```diff
@@ -220,26 +220,26 @@
             "arguments",
             "statements"
         ],
-        // TODO: "curly": true,
-        // TODO: "eofline": true,
+        "curly": false, // changed
```

Comment:
I've never heard you complain about 'curly' before - was this intentional? Does this one annoy you?

It feels like a lack of curly braces is just asking for bugs when copying/pasting, etc. or as described in the main rule [here](https://palantir.github.io/tslint/rules/curly/)
```
if (foo === bar)
    foo++;
    bar++;
```
> In the code above, the author almost certainly meant for both foo++ and bar++ to be executed only if foo === bar. However, they forgot braces and bar++ will be executed no matter what. This rule could prevent such a mistake.

Reply:
I learned that rule a long time ago, but I've never actually seen it occur in the wild.

The main reason for disabling though is just the number of exceptions that are already in the codebase, not wanting to cause a bunch of churn, esp. since there are a lot of customer PRs out.  I'll postpone some of these for reconsideration later, though, since you are right about the auto-correct.